manifests/params.pp

```diff
@@ -473,6 +473,9 @@
       if $::operatingsystem == 'Debian' and $::operatingsystemmajrelease >= '9' {
         $varnetsnmp_owner = 'Debian-snmp'
         $varnetsnmp_group = 'Debian-snmp'
+      } elsif $::operatingsystem == 'Ubuntu' and $::operatingsystemmajrelease >= '18.04' {
```

Comment:
Those facts are legacy and we try to get rid of them in new code + migrate the existing codebase in parallel. Can you please use ```$facts['os']['name']``` and ```$facts['os']['release']['major']```?

----

The tests will fail with that, can you please add a rspec-puppet-facts based test for this?

Reply:
I have rebased to master, I have updated the metadata.

Reply:
Switching params.pp to use the facts hash is much larger change, that may well need doing but is larger in scope than just adding Ubuntu 18.04 support.